buildSrc/src/main/groovy/MuzzlePlugin.groovy

```diff
@@ -73,8 +73,8 @@ class MuzzlePlugin implements Plugin<Project> {
       doLast {
         if (!project.muzzle.directives.any { it.assertPass }) {
           project.getLogger().info('No muzzle pass directives configured. Asserting pass against instrumentation compile-time dependencies')
-          ClassLoader userCL = createCompileDepsClassLoader(project, bootstrapProject)
-          ClassLoader instrumentationCL = createInstrumentationClassloader(project, toolingProject)
+          ClassLoader userCL = createCompileDepsClassLoader(project)
+          ClassLoader instrumentationCL = createInstrumentationClassloader(project)
```

Comment:
almost rid of the bootstrap and tooling project dependencies https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/1225#issuecomment-708342705 🎉